test/functional/ve/standalone/test_f5_base_instance_deploy.py

```diff
@@ -212,14 +240,41 @@ def test_f5_base_instance_deploy_3_nic_11_6(
     check_net_components(bigip, 3)
 
 
-def test_f5_base_instance_deploy_3_nic_12_0(
+def itest_f5_base_instance_deploy_3_nic_12_1(
         HeatStack, symbols, CommonTemplateDir, WaitForLicensedBigIP
 ):
     hc, stack = HeatStack(
         os.path.join(CommonTemplateDir, 'f5_ve_standalone_3_nic.yaml'),
         'func_test_standalone_3_nic',
         parameters={
-            've_image': symbols.ve_12_0_image,
+            've_image': symbols.ve_12_1_image,
+            've_flavor': symbols.ve_flavor,
+            'external_network': symbols.external_network,
+            'mgmt_network': symbols.mgmt_net,
+            'network_1': symbols.data1_net,
+            'network_2': symbols.data2_net,
+            'f5_ve_os_ssh_key': symbols.ssh_key,
+            'admin_password': symbols.bigip_admin_password,
+            'root_password': symbols.bigip_root_password,
+            'license': symbols.license
+        }
+    )
+    updated_stack = hc.stacks.get(stack.id)
+    floating_ip = get_floating_ip_output(updated_stack)
+    bigip = WaitForLicensedBigIP(
+        floating_ip, 'admin', symbols.bigip_admin_password,
+        BIGIP_12_1_VERSION, 3
+    )
+    check_net_components(bigip, 3)
+
+
+def itest_f5_base_instance_deploy_3_nic_13_0(
```

Comment:
With the addition of this method, there should be a change to incorporate:
`293         BIGIP_12_1_VERSION, 3`
swapped to:
`293         BIGIP_13_0_VERSION, 3`
with a s/12_1/13_0/ in this file.  I'm guessing that it's not visible here because git's diff did some reduction behind the scenes that hid it.